docs/yarn.lock

```diff
@@ -0,0 +1,4684 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why there is yarn.lock inside `docs/`? :D

Reply:
:O